packages/core/src/createActor.ts

```diff
@@ -745,11 +778,13 @@ type RequiredOptions<TLogic extends AnyActorLogic> =
 export function createActor<TLogic extends AnyActorLogic>(
   logic: TLogic,
   ...[options]: ConditionalRequired<
+    // Require this...
```

Comment:
perhaps this would be better suited as JSDoc for `ConditionalRequired`?

Reply:
Yeah, true